### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [NickZamiatin/prj-rev-bwfs-tea-cozy#2]
Descriptive alt text should be placed on images, this helps the visually impaired through the use of their screen readers.
